### PR TITLE
Keys

### PR DESCRIPTION
**Pull request by estebanprimost**
- `cli` for keys-db core module
- `keys-db` module using [`level`](https://github.com/Level/level)

### CODE REVIEW

**Comment by dpaez** [comment]
I think this is related to #3, right?

**Comment by estebanprimost** [comment]
> I think this is related to #3, right?

Yes, a kind of

File: packages/cli/src/base-command.js [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
 const { Command, flags } = require('@oclif/command')
 
+const config = require('./config')
+
 class BaseCommand extends Command {
+  get localConfigFolderPath () {
+    return resolve(process.cwd())
+  }
+
+  get globalConfigFolderPath () {
```
Comment (dpaez) [inline packages/cli/src/base-command.js:11]: 👍 for getters

File: packages/cli/src/config.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ module.exports.get = async (key, options = {}) => {
 
   if (!globalConfig && !localConfig) return
 
-  const mergedConfig = deepExtend(globalConfig, localConfig)
+  const mergedConfig = deepExtend(globalConfig || {}, localConfig || {})
```
Comment (dpaez) [inline packages/cli/src/config.js]: Why don't we handle the default case (`{}`) inside `getConfig`?

Comment (estebanprimost) [reply packages/cli/src/config.js]: I need to distinguish between undefined (no file present) or empty config. But probably I can add a default if none on `getConfig`

Comment (estebanprimost) [reply packages/cli/src/config.js]: Solved.

**Review by dpaez — APPROVED** [review]

**Pull request merged**